tools/setup/emoji/build_emoji

```diff
@@ -211,3 +211,12 @@ def dump_emojis(cache_path):
 
 if __name__ == "__main__":
     main()
+
+thumbsup = ["thumbs", "up"]
+th_down = ["thumbs", "down"]
+office_build = ["office", "building"]
+all_dup = [thumbsup, th_down, office_build]
+all_dup_count = all_dup.count()
+
+if all_dup_count > 2:
```

Comment:
Won't this always be true?  If so, I'm not sure why we need the "if" statement here.